api/swagger.yml

```diff
@@ -3783,3 +3800,29 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/ServerError"
+
+  /statistics:
+    post:
+      tags:
+        - statistics
+      operationId: reportUsageEvent
+      summary: report usage event
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UsageEvent"
```

Comment:
I would like to be able to report multiple events in one message.  Not entirely sure why, but most stats reporting APIs that I know of allow collecting this.  The schema change will just be to pass a list of UsageEvents.